recipes/jupyterhub/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "0.6.1" %}
+
+package:
+  name: jupyterhub
+  version: {{ version }}
+
+source:
+  fn: jupyterhub-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/ac/9b/ef3435cf8951d7463f4175a0066a8454e224a598b2159f521cc011f8c2b6/jupyterhub-{{ version }}.tar.gz
+  md5: 6bd795a82bf15a037d097991110b8713
+
+build:
+  number: 0
+  skip: true  # [win or py2k]
+  script: pip install --no-deps .
```

Comment:
@ocefpaf, I have no problem changing the standard if something is more effective, but it would be good if we can get everyone to agree. The issue we have now is if everyone is building each package differently it is more likely some issues creep in or we miss something during a review due to all of the acceptable ways to do something. No one wants broken packages.

> ...because neither package imports setuptools unconditionally.

Ah, ok. Yeah, what we had been doing is a normal install without the options then. Though I agree this is tedious to discover which way to go.